### PR TITLE
feat: bubble chart document section

### PR DESCRIPTION
**Pull request by drake-stamos**

### CODE REVIEW

File: docs/manuals/gui/viselements/charts/bubble-charts.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
+# Bubble Charts
+
+Bubble charts are a type of scatter charts where sizes and colors change depending on the data.
+
+Columns of the data frame will be used to change colors and sizes of the bubbles. The marker parameter is the one giving this information to TaiPy.
+
+## Bubble Chart Key Parameters <a id="table"></a>
+Explanation of parameters and options relevant to candlestick charts can be found [here](#table). Full details for the Taipy Chart control can be found in the Chart Control section of the [Taipy GUI User Manual](https://docs.taipy.io/en/latest/manuals/gui/viselements/chart/)
```
Comment (dinhlongviolin1) [inline docs/manuals/gui/viselements/charts/bubble-charts.md:8]:
```suggestion
Explanation of parameters and options relevant to bubble charts can be found [here](#table). Full details for the Taipy Chart control can be found in the Chart Control section of the [Taipy GUI User Manual](https://docs.taipy.io/en/latest/manuals/gui/viselements/chart/)
```

File: docs/manuals/gui/viselements/charts/bubble-charts.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+marker = {"size": "Sizes", "color": "Colors"}
+
+md = """
+<|{data}|chart|x=x|y=y|marker={marker}|mode=markers|>
+"""
+
+Gui(md).run()
+```
+
+![Simple Bubble Chart with Pandas](simple-bubble-chart-l.png)
```
Comment (dinhlongviolin1) [inline docs/manuals/gui/viselements/charts/bubble-charts.md:43]: Update to use both light and dark images

File: docs/manuals/gui/viselements/charts/bubble-charts.md [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+    "xaxis": {"gridcolor": "#444"},
+}
+
+
+md = """
+<|{data}|chart|x=x|y=y|marker={marker}|mode=markers|layout={layout}|>
+"""
+```
+
+![Marker Size on Bubble Charts](marker-size-bubble-chart-l.png)
```
Comment (dinhlongviolin1) [inline docs/manuals/gui/viselements/charts/bubble-charts.md:77]: same thing

File: docs/manuals/gui/viselements/charts/bubble-charts.md [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+    "showLegend": False,
+}
+
+
+md = """
+<|{data}|chart|x=x|y=y|marker={marker}|mode=markers|layout={layout}|text=text|>
+"""
+```
+
+![Hover Text on Bubble Charts](hover-text-bubble-chart-l.png)
```
Comment (dinhlongviolin1) [inline docs/manuals/gui/viselements/charts/bubble-charts.md:114]: same thing

File: docs/manuals/gui/viselements/charts/bubble-charts.md [hunk 5]
```diff
@@ -0,0 +1,306 @@
...
+    "showLegend": False,
+}
+
+
+md = """
+<|{data}|chart|x=x|y=y|marker={marker}|mode=markers|layout={layout}|>
+"""
+```
+
+![Bubble chart with symbols](bubble-chart-with-symbols-l.png)
```
Comment (dinhlongviolin1) [inline docs/manuals/gui/viselements/charts/bubble-charts.md:151]: same thing

File: docs/manuals/gui/viselements/charts/bubble-charts.md [hunk 6]
```diff
@@ -0,0 +1,306 @@
...
+    "showLegend": False,
+}
+
+
+md = """
+<|{data}|chart|mode=markers|layout={layout}|properties={props}|>
+"""
+```
+
+![Bubble Size Scaling on Charts](bubble-size-scaling-l.png)
```
Comment (dinhlongviolin1) [inline docs/manuals/gui/viselements/charts/bubble-charts.md:226]: same thing

File: docs/manuals/gui/viselements/charts/bubble-charts.md [hunk 7]
```diff
@@ -0,0 +1,306 @@
...
+    "showLegend": False,
+}
+
+
+md = """
+<|{data}|chart|type=scatter|mode=markers|layout={layout}|properties={props}|>
+"""
+```
+
+![Marker Size, Color, and Symbol as an Array](marker-size-color-symbol-array-l.png)
```
Comment (dinhlongviolin1) [inline docs/manuals/gui/viselements/charts/bubble-charts.md:306]: same thing

**Review by dinhlongviolin1 — COMMENTED** [review]
good overall usage

**Comment by FabienLelaquais** [comment]
Replaced by #335.

**Pull request closed without merging**